src/com/core/tictactoe/GamesController.java

```diff
@@ -0,0 +1,38 @@
+package com.core.tictactoe;
+
+public class GamesController {
+
+    CommandLineUI commandLineUI;
+    Game game;
+
+    GamesController(CommandLineUI commandLineUI) {
+        this.commandLineUI = commandLineUI;
+        this.game = null;
+    }
+
+    void gamesMenu() {
+        boolean loopThroughOptions = true;
+        while (loopThroughOptions) {
+            this.commandLineUI.gamingMenu();
+            loopThroughOptions = this.actOnOption(this.getUserOption());
```

Comment:
1. What would happen if you need to validate the user selection?  The way this is written, you'd have to validate within GamesController which doesn't feel like it's responsibility.
2. Returning a boolean from #actOnOption just to control the loop doesn't seem right either.  I feel like it would be better to again think about this polymorphically e.g. I have an Option and I want to 'act' on it... I can explain more in person..

Reply:
1. Validation is done by checking if the user typed a number that was assigned to a game mode (now either 1 or 2). If a user types anything other than 1 or 2, they will be asked again. 
2. Yep, it would be great to discuss it during the Monday IPM (unless you want to talk about it today).

Reply:
P.S. I've simplified it a little bit, but it's still not polymorphism.